rudder-web/src/main/scala/com/normation/rudder/web/services/Section2FieldService.scala

```diff
@@ -156,6 +156,11 @@ class Section2FieldService(val fieldFactory: DirectiveFieldFactory, val translat
 
               valueOpt match {
                 case None =>
+                  varSpec.constraint.default match  {
+                    case None =>
+                    case Some(default) =>
```

Comment:
could you use the syntax:
varSpec.constraint.default.foreach { default => 
setValueForField(default, field, t.from)
}

So that we don't end up with a case branch doing nothing ?